src/test/java/com/exasol/adapter/dialects/scalarfunction/ScalarFunctionsAbstractIT.java

```diff
@@ -252,11 +251,9 @@ void afterAll() {
          */
         @ParameterizedTest
         @MethodSource("getScalarFunctionWithNoParameters")
-        void testFunctionsWithNoParenthesis(final ScalarFunctionCapability function) throws SQLException {
-            runOnExasol(statement -> {
-                assertDoesNotThrow(
-                        () -> statement.executeQuery(this.queryBuilder.buildQueryFor(function.name())).close());
-            });
+        void testFunctionsWithNoParenthesis(final ScalarFunctionCapability function) {
+            runOnExasol(statement -> assertDoesNotThrow(
```

Comment:
Instead of using `assertDoesNotThrow` I would prefer to assert by the result of the query (even as simple as checking if the result set contains some rows)

Reply:
can be improved but not part of this PR

Reply:
please open a ticket then